presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/clientproviders/RetryingPrestoThriftServiceProvider.java

```diff
@@ -48,6 +49,12 @@
     private static final Logger log = Logger.get(RetryingPrestoThriftServiceProvider.class);
     private final PrestoThriftServiceProvider original;
     private final RetryDriver retry;
+    private static final String LIST_SCHEMA_METHOD = "listSchemaNames";
```

Comment:
We can do something like passing a `RetryStats` object to the `RetryDriver.run()` function. And keep a `RetryStats` object per thrift method inside `RetryingPrestoThriftServiceProvider`. By doing this we can get rid of all these string constants and the concurrent hash map.